src/test/java/seedu/address/testutil/TypicalSessions.java

```diff
@@ -67,4 +74,20 @@ public static AddressBook getIntegrationAddressBook() {
     public static List<Session> getTypicalSessions() {
         return new ArrayList<>(Arrays.asList(GETWELL, MACHOMAN));
     }
+
+    /**
+     * Returns an {@code AddressBook} with all the typical Sessions + a Session with interval start that dynamically
+     * to the day after application is loaded.
+     */
+    public static AddressBook getTypicalWithDayAfterAddressBook() {
+        AddressBook ab = new AddressBook();
+        for (Session session : getTypicalWithDayAfterSessions()) {
+            ab.addSession(session);
+        }
+        return ab;
+    }
+
+    public static List<Session> getTypicalWithDayAfterSessions() {
+        return new ArrayList<>(Arrays.asList(GETWELL, MACHOMAN, MACHOMAN_TOMORROW));
+    }
```

Comment:
Is it possible to just merge this together with the getTypicalAddressBook?

Reply:
Not sure if it'll make test cases harder to create in future?